### PR TITLE
updating Maven path for u16java image

### PR DESCRIPTION
**Pull request by Ranjansingh41**
https://github.com/Shippable/pm/issues/12633

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -96,7 +96,7 @@ echo "================ Installing apache-maven $APACHE_MAVEN ================="
 wget http://www-eu.apache.org/dist/maven/maven-3/$APACHE_MAVEN/binaries/apache-maven-$APACHE_MAVEN-bin.tar.gz
 tar xzf apache-maven-$APACHE_MAVEN-bin.tar.gz -C /usr/local && rm -f apache-maven-$APACHE_MAVEN-bin.tar.gz
 ln -fs /usr/local/apache-maven-$APACHE_MAVEN/bin/mvn /usr/bin
-echo 'export PATH=$PATH:/usr/local/apache-maven-$APACHE_MAVEN/bin' >> /etc/drydock/.env
+echo 'M2_HOME=/usr/local/apache-maven-$APACHE_MAVEN/' >> /etc/drydock/.env
```
Comment (rageshkrishna) [inline install.sh]: This is wrong. We both the path and M2_HOME.

Comment (Ranjansingh41) [reply install.sh]: ok i will update

File: install.sh [hunk 2]
```diff
@@ -97,6 +97,7 @@ wget http://www-eu.apache.org/dist/maven/maven-3/$APACHE_MAVEN/binaries/apache-m
 tar xzf apache-maven-$APACHE_MAVEN-bin.tar.gz -C /usr/local && rm -f apache-maven-$APACHE_MAVEN-bin.tar.gz
 ln -fs /usr/local/apache-maven-$APACHE_MAVEN/bin/mvn /usr/bin
 echo 'export PATH=$PATH:/usr/local/apache-maven-$APACHE_MAVEN/bin' >> /etc/drydock/.env
+echo 'M2_HOME=/usr/local/apache-maven-$APACHE_MAVEN/' >> /etc/drydock/.env
```
Comment (deepikasl) [inline install.sh]: echo 'export M2_HOME =/usr/local/apache-maven-$APACHE_MAVEN/' >> /etc/drydock/.env

Comment (Ranjansingh41) [reply install.sh]: updated

**Review by deepikasl — APPROVED** [review]

**Pull request merged**